net/bind/Makefile

```diff
@@ -56,7 +55,7 @@ endef
 define Package/bind-libs
   SECTION:=libs
   CATEGORY:=Libraries
-  DEPENDS:=+libopenssl +zlib
+  DEPENDS:=+libopenssl +zlib +libpthread +(mips||mipsel):libatomic
```

Comment:
That libatomic thing is fascinating... I wonder if it also applies to mips64.

Reply:
Seems gemini needs libatomic as well..